src/view/reminder/appointment-reminder.tsx

```diff
@@ -67,9 +67,10 @@ const AppointmentReminder: FC<AppointmentReminderProps> = (): ReactElement => {
 	}, [playing, audio]);
 
 	useEffect(() => {
-		audio.addEventListener('ended', () => setPlaying(false));
+		const handler = () => setPlaying(false);
```

Comment:
can you replace the handler with callback function?

Reply:
Ok

Reply:
It's not needed inside a useEffect